db/data_migration/20160208121238_republish_statistics_announcements_to_publishing_api.rb

```diff
@@ -0,0 +1,4 @@
+StatisticsAnnouncement.where.not(content_id: "").find_each(batch_size: 100) do |statistics_announcement|
+  statistics_announcement.publish_to_publishing_api
```

Comment:
We might need to tweak that to include the unpublished editions, so it would become: 
```ruby
statistics_announcements = StatisticsAnnouncement.unscoped.where.not(content_id: "")
DataHygiene::PublishingApiRepublisher.new(statistics_announcements)
```